build.gradle

```diff
@@ -135,6 +135,7 @@ dependencies {
     compile 'io.dropwizard.metrics:metrics-core:3.1.0'
     compile 'com.ryantenney.metrics:metrics-spring:3.1.0'
     compile 'io.dropwizard.metrics:metrics-servlets:3.1.0'
+    compile 'org.apache.commons:commons-lang3:3.4'
```

Comment:
I've already added and then removed this dependency because I was using it only for a single method. Now I think we are using it just to compare blank strings. Maybe we could implement this comparison on a Utils class in order not to add another dependency. Wdyt? Maybe from your experience with Java projects you know that we end up using more and more stuff from apache commons.